### PR TITLE
HPCC-16123 Populate return fields

### PR DESCRIPTION
**Pull request by rpastrana**
Populates response fields appropriately (including status).

Signed-off-by: rpastrana rodrigo.pastrana@lexisnexis.com

### CODE REVIEW

**Comment by rpastrana** [comment]
@RussWhitehead please review

**Comment by RussWhitehead** [comment]
Commit message needs work, otherwise looks good

File: esp/scm/ws_esdlconfig.ecm [hunk 1]
```diff
@@ -161,9 +161,10 @@ ESPrequest [nil_remove] ConfigureESDLBindingMethodRequest
 
 ESPresponse [exceptions_inline] ConfigureESDLBindingMethodResponse
 {
-    string ServiceName;         //which ESDL definition are we configuring
-    int ServiceEsdlVersion;     //which version of the ESDL def are we configuring
     string EspProcName;         //Name of ESP Process
+    string EspBindingName;
+    string EsdlDefinitionID;    //The ESDL definition name.ver
+    string EsdlServiceName;     //which ESDL definition are we configuring
     ESPstruct BaseESDLStatus status;
```
Comment (RussWhitehead) [inline esp/scm/ws_esdlconfig.ecm]: Should these changes be versioned here and checked at runtime?

**Comment by RussWhitehead** [comment]
@rpastrana Please update the commit message and Jira

**Comment by rpastrana** [comment]
@RussWhitehead please review.

File: esp/scm/ws_esdlconfig.ecm [hunk 2]
```diff
@@ -161,10 +161,12 @@ ESPrequest [nil_remove] ConfigureESDLBindingMethodRequest
...
-    string EspBindingName;
-    string EsdlDefinitionID;    //The ESDL definition name.ver
-    string EsdlServiceName;     //which ESDL definition are we configuring
+    string EspProcName;                          //Name of ESP Process
+    [max_ver("1.1")] string ServiceName;         //which ESDL definition are we configuring
+    [max_ver("1.1")] int ServiceEsdlVersion;     //which version of the ESDL def are we configuring
+    [min_ver("1.2")] string EspBindingName;
+    [min_ver("1.2")] string EsdlDefinitionID;    //The ESDL definition name.ver
+    [min_ver("1.2")] string EsdlServiceName;     //which ESDL definition are we configuring
     ESPstruct BaseESDLStatus status;
```
Comment (RussWhitehead) [inline esp/scm/ws_esdlconfig.ecm]: Do you have to wrap the setting of these strings with a version check in the c++ code

Comment (rpastrana) [inline esp/scm/ws_esdlconfig.ecm]: the  pre 1.2 fields were never populated, the 1.2+ fields are always populated in post 1.1 code...

Comment (RussWhitehead) [inline esp/scm/ws_esdlconfig.ecm]: The what will be the behavior if a newer client populates a response from an older server without those fields?

**Comment by rpastrana** [comment]
@RussWhitehead review

**Comment by RussWhitehead** [comment]
@rpastrana Looks good

**Comment by rpastrana** [comment]
@richardkchapman please merge

**Pull request merged**